### PR TITLE
Upgrade security public key

### LINKED ISSUE #31591: PGP key is expired
### Description

The PGP key mentioned in SECURITY.md is expired around 2 weeks ago. 
https://github.com/go-gitea/gitea/blob/main/SECURITY.md#protecting-security-information

### Gitea Version

Not related to issue

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

Not related to issue

### Database

None

### PR DESCRIPTION
**Pull request by lunny**
Fix #31591

### CODE REVIEW

**Review by a1012112796 — COMMENTED** [review]
sadlly,after importing this new public key file, I can not verify new releaed binary file.

image

**Comment by a1012112796** [comment]
> sadlly,after importing this new public key file, I can not verify new releaed binary file.
> 
> image

see https://github.com/a1012112796/test_repo/actions/runs/9853651549/job/27204651180

**Comment by silverwind** [comment]
> sadlly,after importing this new public key file, I can not verify new releaed binary file.

I don't think you are meant to use the security key to verify releases, these are different keys I think.

**Comment by a1012112796** [comment]
> > sadlly,after importing this new public key file, I can not verify new releaed binary file.
> 
> I don't think you are meant to use the security key to verify releases, these are different keys I think.

then, how to verify it? thanks

**Comment by silverwind** [comment]
There is another key for "teabot" I think but I don't think that public key is published anywere currently. I had previously recommended to put it on https://dl.gitea.com/.

**Review by techknowlogick — APPROVED** [review]

**Comment by techknowlogick** [comment]
Yes, this is for when security researchers make reports to the security email. The release key is a different key.

**Review by a1012112796 — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The PGP key is valid until June 24, 2024.
 ```
 Key ID: 6FCD2D5B
 Key Type: RSA
-Expires: 6/24/2024
+Expires: 7/9/2025
```
Comment (wxiaoguang) [inline SECURITY.md:22]: image

**Pull request merged**